content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
+new concepts to the API that we'll cover below.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `TLS`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a native Kubernetes API.
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen [implementations][impl] for popular API
+gateways and service meshes and guides are available to start exploring quickly.
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Getting Started
+
+The Gateway API project itself is a Kubernetes API like `Ingress` and you can
+think of the implementations of it as similar to the implementations for
+`IngressClass` or `StorageClass` in that Kubernetes itself has no default
```

Comment:
I'd make IngressClass and StorageClass be hyperlinks.

Reply:
```suggestion
[IngressClass](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class) or [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) in that Kubernetes itself has no default
```